net-analyzer/openvas-scanner/openvas-scanner-20.8.1.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake flag-o-matic toolchain-funcs
+
+MY_PN="openvas"
+MY_DN="openvassd"
+
+DESCRIPTION="Open Vulnerability Assessment Scanner"
+HOMEPAGE="https://www.greenbone.net/en/"
+SRC_URI="https://github.com/greenbone/openvas-scanner/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2 GPL-2+"
+KEYWORDS="~amd64 ~x86"
+IUSE="cron extras test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	acct-user/gvm
+	app-crypt/gpgme:=
+	dev-db/redis
+	dev-libs/glib
+	dev-libs/libgcrypt:=
+	dev-libs/libksba
+	>=net-analyzer/gvm-libs-20.8.0
+	net-analyzer/net-snmp
```

Comment:
Hm I think this should be changed to net-analyzer/net-snmp:= to fix https://bugs.gentoo.org/733894 .
But here it's also automatically detected, furthermore this can not disabled by configure option.
So I'm not sure if I need to patch here too, to fail if the dependency is not found.

Reply:
Please work on a patch to disable it, but you can do that after this PR gets merged if you've never done it before and need to study/test excessively it.
Examples:
https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Automagic_dependencies#CMake
https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-libs/check/files/check-0.14.0-r2-disable-automagic-dep.patch